EDAM_dev.owl

```diff
@@ -59834,11 +59834,14 @@
     <owl:Class rdf:about="http://edamontology.org/topic_3523">
         <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_3077"/>
         <created_in>1.8</created_in>
-        <oboInOwl:hasDefinition>RNAi experiments.</oboInOwl:hasDefinition>
+        <oboInOwl:hasBroadSynonym>Post-transcriptional regulation</oboInOwl:hasBroadSynonym>
```

Comment:
IMO this broad synonym should be removed, it is comparing a class of biological mechanisms against a class of data acquisition techniques